src/EditorFeatures/CSharpTest/Diagnostics/SimplifyTypeNames/SimplifyTypeNamesTests.cs

```diff
@@ -3755,7 +3755,6 @@ void M()
     }
 }",
                 options: Option(CodeStyleOptions.QualifyPropertyAccess, false, NotificationOption.Warning),
-                diagnosticCount: 1,
```

Comment:
`TestDiagnosticSeverityAndCountAsync` is now `TestDiagnosticInfoAsync`. The earlier version of this method, `TestDiagnosticSeverityAndCountAsync`, checked some _basic_ stuff about all instances of a particular diagnostic, but didn't allow for testing the diagnostic message (which can vary), etc.. The new version checks more detailed information about a _single_ diagnostic. This is implicit in the name, `TestDiagnosticInfoAsync`, which also does a `.Single()` on the retrieved list of diagnostics, so anybody incorrectly using this method to attempt to ensure multiple diagnostics are handled correctly will fail at that point.